code/modules/client/preferences.dm

```diff
@@ -1007,7 +1006,7 @@ NOTE:  The change will take effect AFTER any current recruiting periods."}
 							f_style = facial_hair_styles_list["Shaved"]
 
 						//reset hair colour and skin colour
-						r_hair = 0//hex2num(copytext(new_hair, 2, 4))
+						r_hair = 1//hex2num(copytext(new_hair, 2, 4))
```

Comment:
I think I understand what you intend to do here, but it doesn't seem like a good idea.
What's the code that breaks when this is 0?

Reply:
hold on i think i have a better solution

Reply:
no i dont nevermind

Reply:
wait maybe i do

Reply:
basically because you've had me coopt the human r_hair variable, but the minimum required value is 1 in order for vox hair color to work, that is a Bandaid so people do not receive error sprites when switching from a human with a hair color to a vox, and do not enter the game that way

i can probably just create files for the number 0 so that theyre not error sprites but i think doing the r_hair thing either way probably breaks hair dye working on vox somehow. i don't have the mental fortitude to check because i know something will go horribly wrong and then i have to fix it and i don't want to and that's why i got sonix to write that portion of the code

i'd also like to say that skin tone variables (s_color and s_tone), which exist in a similar dichotomy to v_hair and the 3 rgb_hair variables (with vox/grey skin color being a selection and human skin color being a number you input) are ALSO separate variables and basically this is lazy and not internally consistent, purely to avoid touching the database, which i would do myself if i knew how

![1615525163996](https://user-images.githubusercontent.com/81062934/152667751-f9b41323-01cf-4054-b310-f78a32971e2f.jpg)

Reply:
that's very confusing